test/run-tests

```diff
@@ -636,7 +662,7 @@ def make_html(source_file):
     return html_file
 
 
-def handle_task(gh, args, config, task, dry_run=False):
+def handle_task(gh, args, config, task, ansible_ver_str, dry_run=False):
```

Comment:
```suggestion
def handle_task(gh, args, config, task, ansible_version, dry_run=False):
```

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0